### ISSUE TITLE
No error message with WithCheck()

### ISSUE THREAD

**Issue by JustinDroege** [PragmaticFlow/NBomber#459]
I have the same error like in this issue: https://github.com/PragmaticFlow/NBomber/issues/426
But im using the newest version from NBomber and the HTTP plugin (2.1.3 & 2.0.1)

Here is my code:
```
var step = Step.Create("step", clientFactory: httpClientFactory, feed: circularFeed, async context =>
{
    var request =
        Http.CreateRequest("POST", "https://localhost")
            .WithBody(new StringContent(context.FeedItem))
            .WithCheck(async (response) => 
                response.IsSuccessStatusCode
                    ? Response.Ok(statusCode: (int)response.StatusCode)
                    : Response.Fail(error: await response.Content.ReadAsStringAsync(), statusCode: ((int)response.StatusCode))
            );
    var response = await Http.Send(request, context);
    if (response.IsError) // this is true when there is an error
        context.Logger.Error(response.ErrorMessage); //There is an empty string
    return response;
}, timeout: TimeSpan.FromSeconds(60));
```

**Comment by Smrecz** [comment]
The version with the fix is not yet released.

@AntyaDev can we have NBomber.Http release with that fix any time soon? That's a really annoying one ;)

**Comment by JustinDroege** [comment]
Thanks for the answer. Maybe i will give a look and open up a pull request. When its written in F#, it will be my first time :D

**Comment by AntyaDev** [comment]
Hi guys,

Thanks for informing me about this issue.
I published v3.0.0-beta2 with this fix
